api/primary.go

```diff
@@ -149,11 +150,12 @@ func setupPrimaryRouter(r *mux.Router, context *context, enableCors bool) {
 				if enableCors {
 					writeCorsHeaders(w, r)
 				}
+				context.apiversion = mux.Vars(r)["version"]
 				localFct(context, w, r)
 			}
 			localMethod := method
 
-			r.Path("/v{version:[0-9.]+}" + localRoute).Methods(localMethod).HandlerFunc(wrap)
+			r.Path("/v{version:[0-9]+.[0-9]+}" + localRoute).Methods(localMethod).HandlerFunc(wrap)
```

Comment:
@dongluochen I couldn't find one. We don't seem to be using this version anywhere in Swarm so far, I believe it's only meant for the API to mimic that of Engine. What do you suggest?